src/platform/imx8ulp/include/platform/lib/cpu.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 NXP Corporation. All rights reserved.
```

Comment:
`Copyright(c) 2021 NXP` I believe is enough.
Keep it  as in the other files

Reply:
@TangleZ 

Current guidelines from NXP is to use:

```
  * Copyright 2021 NXP
  ```